src/Card/CardHeader.js

```diff
@@ -33,6 +39,10 @@ export type Props = {
    * The Avatar for the Card Header.
    */
   avatar?: Node,
+  /**
+   * The action to display in the card header.
+   */
+  action?: Node,
```

Comment:
I'm adding the missing TypeScript counter part.

Reply:
Thanks!